uaclient/cli.py

```diff
@@ -41,6 +44,21 @@
 
 STATUS_FORMATS = ["tabular", "json", "yaml"]
 
+UA_LOGS = (
```

Comment:
Most of these paths are already stored on `UAConfig`. Furthermore, some of these paths can be overridden in the config. So we should rely on the `UAConfig` object to obtain those paths

Reply:
Good, well noted. Will change it.

Reply:
I think we also want /etc/ubuntu-advantage/uaclient.conf too.